### DISCUSSION TITLE
Input filter does not allow to delete the entered character

### DISCUSSION THREAD

**Discussion by Legolaz7** [flet-dev/flet#3926] [Q&A] [answered]
I copied the code here https://flet.dev/docs/controls/textfield/#input_filter
I set the input filter so that only "numbers" and "dot" values ​​are accepted
I enter numbers, and it does not allow me to delete the first value.
The code itself:
```
import flet as ft

def main(page: ft.Page):
    page.window.width = 400

    def textbox_changed(e):
        t.value = e.control.value
        page.update()

    t = ft.Text()
    tb = ft.TextField(
        label="Textbox with 'change' event:",
        on_change=textbox_changed,
        input_filter=ft.InputFilter(regex_string=r"[0-9.]")
    )

    page.add(tb, t)

ft.app(main)
```

**Comment by Legolaz7** [comment]
If you remove input _filter, everything works.

**Comment by ndonkoHenri** [comment, marked as answer]
Read [this](https://medium.com/python-in-plain-english/whats-new-in-flet-v0-24-0-83f758245f4d#:~:text=TextField-,InputFilter,-We%20modified%20how) concerning the latest Flet v0.24.0 release.

**Comment by Legolaz7** [comment]
I can't follow the link, there's some kind of glitch(

**Reply by ndonkoHenri** [reply]
![image](https://github.com/user-attachments/assets/bd7255db-ef52-4b7e-99ef-5c220304fe50)
